stdlib/typing.pyi

```diff
@@ -50,7 +50,7 @@ Type: _SpecialForm = ...
 ClassVar: _SpecialForm = ...
 if sys.version_info >= (3, 8):
     Final: _SpecialForm = ...
-    def final(f: _F) -> _F: ...
```

Comment:
`@final` returns the original function, so I think using `ParamSpec` would be less correct.